src/pages/App.tsx

```diff
@@ -66,6 +61,72 @@ const Section = styled.section`
     margin-bottom: 50px;
     margin-top: 50px;
   }
+
+  h1 {
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 56px;
+    color: ${colors.secondaryFontColor};
+  }
+
+  h2 {
+    font-weight: 700;
+    font-size: 28px;
+    line-height: 36px;
+    color: ${colors.mainFont};
+  }
+
+  h3 {
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 56px;
+    color: ${colors.mainFont};
+  }
+
+  h4 {
+    font-family: ${fonts.lora};
+    font-weight: 400;
+    font-style: italic;
+    font-size: 24px;
+    line-height: 28px;
+    letter-spacing: 0.01em;
+    color: ${colors.italicColor};
+  }
+
+  h5 {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  h6 {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  p {
+    font-family: ${fonts.mainFont};
+    font-weight: 400;
+    size: 16px;
+    line-height: 24px;
+    letter-spacing: 0.5px;
+  }
+
+  .button {
```

Comment:
Where are `.caption` and `.button` being used?

Reply:
They will be used in future Components. We have two future Des cases in pending dev

Reply:
@ajkendal We'll want to make styled components for these. We also have a shared button component in the works that may prove useful to expand on (@HereComePretzel is working on that). Let's leave these out of this PR, and then create the generic styled components we need to when they're used in the future PR

Reply:
@bestfoodalex 

Would it not be best to have them in there for generics, so when they do update these cases and have all the Design Standards that @alison-voghel  has laid out, they are in the standard files, and dont need to be redone for each component.

Reply:
@ajkendal The components would be generic, but would help us build a component library, rather than pigeonholing ourselves with styles so specific to tags (causing issues like the heading hierarchy one). In other words, the broader goal will be to import our components and use them rather than rely on a more traditional global CSS approach. I.e., we want to go the styled-components route. If you're more comfortable doing it this way, we can merge this PR and I'll do the work to make the title components and open a PR so you can see what I mean.

I'd still prefer we not add any styles that are not being used yet, though - just so the code we add in an individual PR is meaningful (so since the button and caption classes aren't being used yet, we'd save those for the PR where they are used)

Reply:
@bestfoodalex  I just moved everything to a generic global style in theme, that can be updated or modified at later time